src/lib/alarms/trade_offer.ts

```diff
@@ -6,6 +6,24 @@ import {AnnotateOffer} from '../bridge/handlers/annotate_offer';
 import {PingCancelTrade} from '../bridge/handlers/ping_cancel_trade';
 import {CancelTradeOffer} from '../bridge/handlers/cancel_trade_offer';
 import {FetchSteamUser} from '../bridge/handlers/fetch_steam_user';
+import {rgDescription} from '../types/steam';
+import {HasPermissions} from '../bridge/handlers/has_permissions';
+
+export interface ExtendedOfferStatus {
+    offer_id: string;
+    state: TradeOfferState;
+    given_asset_ids?: ExtendedSingleOffer[];
```

Comment:
`asset_id` isn't the right name for this since it actually is a list of item attributes including the asset ID.

`given_assets` or `given_items` would have better meaning.